### PR TITLE
.version file entries

### LINKED ISSUE #30689: [Breaking change]: .NET SDK .version file will now include the specific build version
### Description

Today, the .NET SDK installation folder (under dotnet\sdk\<version>\) contains a .version file. This file includes the hash for the commit for that version of the .NET SDK, the stable version, and the RuntimeIdentifier of the .NET SDK.

This change introduced in 6.0.401 will add the specific build number from our build system. This was done by request as the simpler, short build version cannot be used by the dotnet-install script prior to release.

**Example**
baae4cac8b288405720786a3bcd35ee5a6086f1b
6.0.401
win-x64
_6.0.401-servicing.22414.4_

### Version

Other (please put exact version in description textbox)

### Previous behavior

dotnet\sdk\<version>\.version file contained three values:

git commit hash
version
RuntimeIdentifier

### New behavior

dotnet\sdk\<version>\.version file contains four values:
Git commit hash
Simplified Version
RuntimeIdentifier
Precise build Version

### Type of breaking change

- [ ] **Binary incompatible**: Existing binaries may encounter a breaking change in behavior, such as failure to load/execute or different run-time behavior.
- [ ] **Source incompatible**: Source code may encounter a breaking change in behavior when targeting the new runtime/component/SDK, such as compile errors or different run-time behavior.

### Reason for change

To enable internal testing of the dotnet-install script

### Recommended action

No action required unless you have tooling that parses the .version file. Then you will have to account for a four entry in that file.

### Feature area

SDK

### Affected APIs

_No response_

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #30689

### CODE REVIEW

File: docs/core/compatibility/sdk/6.0/version-file-entries.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Breaking change: .version file includes build version"
+description: Learn about a breaking change in the .NET SDK where the .version file now includes the specific build version.
+ms.date: 09/16/2022
+---
+# .version file includes build version
+
+The .NET SDK installation folder (under *dotnet\\sdk\<version>*) contains a *.version* file. This file includes the following data:
```
Comment (IEvangelist) [inline docs/core/compatibility/sdk/6.0/version-file-entries.md:8]:
```suggestion
The .NET SDK installation folder (under *dotnet/sdk/<version>*) contains a *.version* file. This file includes the following data:
```

The backslash is Windows, whereas the forward slash is both Unix and Windows. In other words, to be cross-platform, use the forward slash.

File: docs/core/compatibility/sdk/6.0/version-file-entries.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+6.0.401
+win-x64
+6.0.401-servicing.22414.4
+```
+
+## Previous behavior
+
+The *dotnet\\sdk\<version>.version* file contained three values:
+
+- git commit hash
```
Comment (IEvangelist) [inline docs/core/compatibility/sdk/6.0/version-file-entries.md:29]:
```suggestion
- Git commit hash
```

**Review by IEvangelist — APPROVED** [review]
Looks good, a few nits.

File: docs/core/compatibility/sdk/6.0/version-file-entries.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+description: Learn about a breaking change in the .NET SDK where the .version file now includes the specific build version.
+ms.date: 09/16/2022
+---
+# .version file includes build version
+
+The .NET SDK installation folder (under *dotnet\\sdk\<version>*) contains a *.version* file. This file includes the following data:
+
+- The hash for the commit for that version of the .NET SDK.
+- The stable version.
+- The RuntimeIdentifier of the .NET SDK.
```
[the remark this answers is not in the document]
Comment (gewarren) [reply docs/core/compatibility/sdk/6.0/version-file-entries.md]: I don't actually think linking to that property is correct here - the property identifies the platform "of the currently executing process". Instead I linked to our RID catalog article.

**Pull request merged**